### PR TITLE
feat: Added all components and pages

### PR DESCRIPTION
**Pull request by TusharJade**
### Added all components and pages with routing.

Just the landing page is remaining will make it in the next PR.

Preview link - https://deploy-preview-2--khariodo-the-ecom-app.netlify.app/

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,30 @@
...
+import { Navbar } from "./components/Navbar/Navbar";
+import {
+  AddToCartPage,
+  LandingPage,
+  LoginPage,
+  ProductListing,
+  SignUpPage,
+  WishlistPage,
+} from "./pages/index";
+import Mockman from "mockman-js";
```
Comment (Koushith) [inline src/App.js:11]: you can move the library imports to the top.

**Review by Koushith — APPROVED** [review]
folder structure and naming LGTM. good to go.

File: src/App.js (hunk 1 above)
Comment (TusharJade) [reply src/App.js:11]: Yes will move it to top.

**Review by SumitSaundadkar — APPROVED** [review]
Everything looks great now work on eccom using cl...good to go.

**Pull request merged**